documentation/examples/11-easing/00-easing/Controls.svelte

```diff
@@ -28,6 +28,7 @@
 		{:else}
 			<ul>
 				{#each [...eases] as [name]}
+				<!-- svelte-ignore a11y-click-events-have-key-events a11y-no-noninteractive-element-interactions -->
```

Comment:
instead of ignoring the warnings here, nest a button inside the `<li>` instead and apply the click handler there. e.g.

```html
<li><button on:click={() => (current_ease = name)}>{name}</button></li>
```

(note: this may require adjusting some styles)